### PR TITLE
Fix Android builds when gradle args and project dir have spaces

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
In case you try building application, it will fail in case the following conditions are applied:
* build on Windows
* build for Android
* build in release
* path to project dir has spaces
* path to certificate (keystore) has spaces OR the certificate(keystore) name itself has spaces OR the password has spaces

The problem is in Node.js's `child_process` when `spawn` is used - on Windows it has issues when both the command (in our case this is the FULL PATH to gradlew.bat executable in platforms dir of the project) and ANY of the arguments passed to `child_process's spawn` have spaces.
As we are also setting the current working directory of the spawned process to be the path to `<project dir>/platforms/android`, change the `command` passed to `spawn` to be just `gradlew`. This way we'll never have spaces in the command and we'll not hit this issue.

More information for the Node.js issue: https://github.com/nodejs/node-v0.x-archive/issues/25895

There's a similar issue (linked in the comments of the above) for newer node.js versions. The suggested solution there is to use `{ shell: true}` parameter to `spawn` method, but this will require handling of all spaces in the args on our own.

### CODE REVIEW

File: lib/services/android-project-service.ts [hunk 1]
```diff
@@ -417,24 +415,14 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 	}
 
 	public stopServices(projectRoot: string): Promise<ISpawnResult> {
-		let gradleBin = path.join(projectRoot, "gradlew");
-		if (this.$hostInfo.isWindows) {
-			gradleBin += ".bat";
-		}
-
-		return this.$childProcess.spawnFromEvent(gradleBin, ["--stop", "--quiet"], "close", { stdio: "inherit", cwd: projectRoot });
+		// TODO: Check if we have to pass compileSdk and other options here
```
Comment (yyosifov) [inline lib/services/android-project-service.ts]: can we help with this TODO?

File: lib/services/android-project-service.ts [hunk 2]
```diff
@@ -504,6 +492,19 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 
 		return versionInManifest;
 	}
+
+	private async executeGradleCommand(projectRoot: string, gradleArgs: string[], childProcessOpts?: SpawnOptions, spawnFromEventOptions?: ISpawnFromEventOptions): Promise<ISpawnResult> {
+		const gradleBin = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
```
Comment (yyosifov) [inline lib/services/android-project-service.ts]: I would rename this const to "gradlew" as it points to the wrapper, not gradle itself.

File: lib/services/android-project-service.ts (hunk 1 above)
Comment (rosen-vladimirov) [inline lib/services/android-project-service.ts]: ping @Plamen5kov , @Pip3r4o  for this

Comment (rosen-vladimirov) [inline lib/services/android-project-service.ts]: I'm not sure why we need to call stop and should we pass other options...

Comment (rosen-vladimirov) [inline lib/services/android-project-service.ts]: I've just tested it without args and it's working fine, so I'll remove the TODOD

**Pull request merged**